go.mod

```diff
@@ -29,7 +29,7 @@ require (
 	github.com/fsnotify/fsnotify v1.5.1
 	github.com/getsentry/sentry-go v0.11.0
 	github.com/go-asn1-ber/asn1-ber v1.5.3 // indirect
-	github.com/go-morph/morph v0.0.0-20211202070306-dbdc0736c17e
+	github.com/go-morph/morph v0.0.0-20220110203813-7e65a95885ea
```

Comment:
Hmm .. why not a tagged release? Looks neater :P

Reply:
Agreed, l'll update it once we roll out a new version.